install.bat

```diff
@@ -127,8 +127,10 @@ if %ERRORLEVEL% neq 0 (
 echo Downloading latest model
 IF NOT EXIST models (
     md models
-) else ()
-powershell -Command "Invoke-WebRequest -Uri 'https://the-eye.eu/public/AI/models/nomic-ai/gpt4all/gpt4all-lora-quantized-ggml.bin' -OutFile 'models/gpt4all-lora-quantized-ggml.bin'"
+)
```

Comment:
Can you please change this so that if the model exists, the user is prompted if he wants to override the model or not.

Reply:
Added a (y/n) prompt

![cmd_jCD1Ud7W9W](https://user-images.githubusercontent.com/81461845/230576262-238c345e-b8eb-4fed-9988-d02e6b917bf6.png)